platform/mime/core/platform-mime-core-api/src/main/java/ddf/mime/MimeTypeResolver.java

```diff
@@ -17,7 +17,7 @@
  * Interface defining an OSGi service that can map a list of file extensions to their corresponding
  * mime types, and vice versa.
  *
- * @author Hugh Rodgers, Lockheed Martin
+ * @author Hugh Rodgers
  * @author ddf.isgs@lmco.com
```

Comment:
Did you miss this `ddf.isgs@lmco.com`? Seems like you were removing them all except this one and the one in `TikaMimeTypeResolver.java`